_posts/2021-09-23-reassign-partitions.md

```diff
@@ -0,0 +1,282 @@
+---
+layout: post
+title:  "Reassign partitions in Kafka Cluster"
+date: 2021-09-23
+author: shubham_rawat
+---
+
+Scaling Kafka Brokers, removing old brokers, and increasing the number of topics and partitions are all pretty common tasks for Kafka users.
+All these cases can cause the load on the brokers to get unbalanced. The most convenient way to manage an unbalanced cluster is to reassign the partitions among all the brokers.
```

Comment:
Do we really want to put the focus on cluster balancing here? We have Cruise Control and KafkaRebalance for this and we should point users to that and not to the manual reassignment.